modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,111 @@
-In the current release, you must create a Windows template and a custom ConfigMap to import a RHV VM with a Windows operating system. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ#1856654] for details.
+If you are importing a Windows virtual machine, you must also create a Windows template. link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[(*BZ#1856654*)]
 ====
 
 .Procedure
 
-. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>` as in the following example:
+. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>`. The operating system appears in the `<os>` section of the XML output, as shown in the following example:
 +
 [source,xml]
 ----
 <os>
 ...
-<type>windows_2016x64</type>
+<type>rhel_8x64</type>
 </os>
 ----
 
-. Obtain a list of the common {VirtProductName} operating system templates by running the following command on your {product-title} cluster:
+. To import a Linux VM:
+
+.. Obtain a list of the {product-title} templates:
 +
 ----
 $ oc get templates -n openshift --show-labels | tr ',' '\n' | grep os.template.kubevirt.io | sed -r 's#os.template.kubevirt.io/(.*)=.*#\1#g' | sort -u
 ----
 +
-If the RHV VM operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap.
+The output is similar to the following example:
++
+----
+fedora31
+fedora32
+...
+rhel8.1
+rhel8.2
+...
+----
++
+If the RHV VM operating system appears in this template list, you do not need to create a ConfigMap.
+
+.. If the RHV VM operating system does not appear in the template list, create a ConfigMap for a common template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default <1>
+data:
+  guestos2common: |
+    "Red Hat Enterprise Linux Server": "rhel" <2>
+    "CentOS Linux": "centos"
+    "Fedora": "fedora"
+    "Ubuntu": "ubuntu"
+    "openSUSE": "opensuse"
+  osinfo2common: |
+    "<operating-system>": "<common-template>" <3>
+EOF
+----
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify the Linux operating system.
+<3> Specify the Linux operating system version and the {product-title} common template that it maps to:
+
+* `"rhel_6_9_plus_ppc64": "rhel6.9"`
+* `"rhel_6_ppc64": "rhel6.9"`
+* `"rhel_6": "rhel6.9"`
+* `"rhel_6x64": "rhel6.9"`
+* `"rhel_7_ppc64": "rhel7.7"`
+* `"rhel_7_s390x": "rhel7.7"`
+* `"rhel_7x64": "rhel7.7"`
+* `"rhel_8x64": "rhel8.1"`
+* `"sles_11_ppc64": "opensuse15.0"`
+* `"sles_11": "opensuse15.0"`
+* `"sles_12_s390x": "opensuse15.0"`
+* `"ubuntu_12_04": "ubuntu18.04"`
+* `"ubuntu_12_10": "ubuntu18.04"`
+* `"ubuntu_13_04": "ubuntu18.04"`
+* `"ubuntu_13_10": "ubuntu18.04"`
+* `"ubuntu_14_04_ppc64": "ubuntu18.04"`
+* `"ubuntu_14_04": "ubuntu18.04"`
+* `"ubuntu_16_04_s390x": "ubuntu18.04"`
+
```

Comment:
That's because they are not going to use the existing Windows mapping for 2.4.

Reply:
Users must know the existing mapping including Windpws mapping. 
The windows w/a runs over it, but once this w/a will be removed, the Windows mapping will be valid.
It may change in coming CNV versions, so need to track:
https://github.com/kubevirt/vm-import-operator/blob/master/pkg/os/os-mapper-provider.go#L102

Reply:
What is "w/a"?

Reply:
I'm afraid that it will be confusing to the user if we provide both mapping for existing Windows templates AND we tell the user to create a Windows template. The user won't know that the mapping here is not for 2.4.0, only for 2.4.1. Couldn't we comment out the Windows mappings until the Windows disk bug is fixed?